src/config.rs

```diff
@@ -10,6 +10,7 @@ pub struct Config {
     pub editor_args: Option<Vec<String>>,
     pub temporary_project_dir: PathBuf,
     pub git_repo_depth: Option<Depth>,
+    pub vcs: Option<bool>,
```

Comment:
You should take an arbitrary string and pass it to cargo as is if specified

Reply:
To allow the user to choose another vcs 
Not a bad idea, lemme check this